### ISSUE TITLE
[java] UnusedImports when @see / @link pattern includes a FQCN

### ISSUE THREAD

**Issue by jsotuyod** [pmd/pmd#2025]
**Affects PMD Version:** 6.18.0

**Rule:** bestpractices/UnusedImports

**Description:** If the class mentioned in a `@see` or `@link` is a FQCN, the parameters are not analyzed and may be considered unused imports.

**Code Sample demonstrating the issue:**

```java
import spark.Request; // flaged, should not
import spark.Response; // flaged, should not

/**
 * {@link foo.bar.MyController#startTransaction(Request)}
 * @see foo.bar.MyController#startTransaction(Response)
 */
public class VendingV2PaymentRequest {
}	
```

**Running PMD through:** Any
